common/src/main/java/io/novafoundation/nova/common/view/SegmentedTabLayout.kt

```diff
@@ -0,0 +1,67 @@
+package io.novafoundation.nova.common.view
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Path
+import android.graphics.drawable.InsetDrawable
+import android.util.AttributeSet
+import com.google.android.material.tabs.TabLayout
+import io.novafoundation.nova.common.R
+import kotlin.math.roundToInt
+
+class SegmentedTabLayout @JvmOverloads constructor(
```

Comment:
We already have similar ui component on "Vote" tab. Can you please check & maybe adapt it? I dont think we should keep multiple implementations of the same thing

Reply:
I think we can adapt it in Vote screen, since SegmentedTabLayout supports animation and integration with view pager